pages/pil/dashboard/index.js

```diff
@@ -84,6 +102,9 @@ module.exports = settings => {
       res.locals.static.skipTraining = get(req.session, [req.profileId, 'skipTraining'], null);
       res.locals.static.isAsru = req.user.profile.asruUser;
       res.locals.static.isLicensing = req.user.profile.asruLicensing;
+      res.locals.static.canTransferPil = req.pil.status === 'active' && req.user.profile.id === req.profile.id; // can only transfer own (active) pil
+
+      // res.locals.static.canTransferPil = false; // PIL TRANSFER FEATURE FLAG (disables edit establishment link)
```

Comment:
assume this will be removed before this PR is un-WIPed?

Reply:
@lennym requested that we had an easy way to disable the UI for this feature so that we can still test it on prod without having the link actually visible - I've commented it out as we might want to leave it on for S&T demo if it makes the cut?

Reply:
OIC - carry on then